packages/addons/service/argononecontrol/package.mk

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2025-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="argononecontrol"
+PKG_VERSION="1.1.6"
+PKG_SHA256="9c15299d48efe5a138ce3ce8d0b78bb7d2f5317988e67346295629af752967ee"
+PKG_REV="0"
+PKG_ARCH="arm aarch64"
```

Comment:
Change to `PGK_ARCH="aarch64" because for LE13 (and LE12) RPi4/5 no longer use `arm` userspace.